### PR TITLE
MTD Validation: Update Btl Local Reco Monitoring

### PR DESCRIPTION
**Pull request by ctarricone**
#### PR description:

This PR aims at updating Btl local reco validation, adding monitor elements for cluster local Y resolution used to test the scenario D95 for the release CMSSW_13_0_0_pre4. Checks are separately done for global Z < 0 and Z > 0.   

#### PR validation:

Code compiles, runs, and it has been used for the validation of the aforementioned campaign.

### CODE REVIEW

**Comment by fabiocos** [comment]
@ctarricone I see the new histograms added, but I do not see the old, combined stuff, removed, am I correct?

**Review by fabiocos — COMMENTED** [review]
When removing the plots for all z, please remove also the ```MonitorElement``` declaration and definition

File: Validation/MtdValidation/plugins/BtlLocalRecoValidation.cc [hunk 1]
```diff
@@ -453,25 +477,77 @@ void BtlLocalRecoValidation::analyze(const edm::Event& iEvent, const edm::EventS
...
         float ylocal_res = local_point.y() - cluLocalPosSIM.y();
+
         float z_res = global_point.z() - cluGlobalPosSIM.z();
 
         meCluZRes_->Fill(z_res);
 
         if (matchClu && comp != nullptr) {
           meCluLocalXRes_->Fill(xlocal_res);
           meCluLocalYRes_->Fill(ylocal_res);
+
```
[the remark this answers is not in the document]
Comment (fabiocos) [reply Validation/MtdValidation/plugins/BtlLocalRecoValidation.cc:488]: sorry, just YRes I mean

File: Validation/MtdValidation/plugins/BtlLocalRecoValidation.cc [hunk 2]
```diff
@@ -453,25 +477,77 @@ void BtlLocalRecoValidation::analyze(const edm::Event& iEvent, const edm::EventS
...
+          if (global_point.z() > 0) {
+            meCluLocalYResZGlobPlus_->Fill(ylocal_res);
+            if (optionalPlots_) {
```
[the remark this answers is not in the document]
Comment (ctarricone) [reply Validation/MtdValidation/plugins/BtlLocalRecoValidation.cc:491]: just to clarify, I maintain histos with local Y pull splitted in Z > 0 and Z < 0  (even as non-optional), while I completely remove the old local Y pull histogram (w/ all the glob Z merged) as you commented below. Am I correct?

**Comment by fabiocos** [comment]
please test

**Comment by emanueleusai** [comment]
+1

- Differences in HLT/SiStrip/ControlView are unrelated

**Comment by perrotta** [comment]
+1

**Pull request merged**